test/performance/Makefile.am

```diff
@@ -95,6 +95,8 @@ odp_ipsecfwd_SOURCES = odp_ipsecfwd.c
 AM_CFLAGS += $(LIBCONFIG_CFLAGS)
 endif
 
+PRELDADD += $(LIBBENCH_COMMON)
```

Comment:
For safety, the public bench functions could have a some harder-to-name-clash prefix at some point.

Reply:
True, will keep this in mind.